pkg/controllers/multiclusterservice/mcs_controller.go

```diff
@@ -197,6 +198,11 @@ func (c *MCSController) deleteMultiClusterServiceWork(mcs *networkingv1alpha1.Mu
 			continue
 		}
 
+		if err := c.cleanProvisionEndpointSliceWork(work.DeepCopy()); err != nil {
```

Comment:
This function deletes mcs work and eps work,  should we renames it such as deleteMCSAndEpsWork?

Reply:
EndpointSlice work is the derived resource of mcs work, I think it's ok to use this name

Reply:
ok